staging/src/k8s.io/component-helpers/OWNERS

```diff
@@ -0,0 +1,26 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- lavalamp
+- smarterclayton
+- deads2k
+- sttts
+- liggitt
+- caesarxuchao
+reviewers:
+- thockin
+- lavalamp
+- smarterclayton
+- wojtek-t
+- deads2k
+- derekwaynecarr
+- caesarxuchao
+- cheftako
+- mikedanese
+- liggitt
+- gmarek
+- sttts
+- ncdc
+- logicalhan
+labels:
+- sig/api-machinery
```

Comment:
i'd recommend starting with a smaller list of owners - e.g. those who "championed" the repository creation idea.
instead of the full list from k/k, as it might feel a bit odd to get pinged by the bots for review on a new repository that you did not know existed?

Reply:
Good point.

@deads2k @soltysh @liggitt @dims would you be interested in being approvers and reviewers?

Reply:
+1 i can help.

Reply:
that seems ok, though I'd like to see clear top-level packages with sig-specific reviewers/approvers/labels (and drop the root sig/api-machinery label) so that very little actually falls through to the root module approvers

Reply:
> @deads2k @soltysh @liggitt @dims would you be interested in being approvers and reviewers?

Yes, I can help here.  I'd like to see the bar be reasonably high.

Reply:
> I'd like to see clear top-level packages with sig-specific reviewers/approvers/labels

Can you more elaborate on that? Are you saying `k8s.io/component-helpers/scheduling` naming is not sufficient or are you referring to a rule to follow for contributors?

Reply:
More of a rule to follow (e.g. adding the expected structure to the readme, where each top-level package has a clear sig-oriented reviewer/approver list, and no code lives in the top-level package)

Reply:
I agree with Jordan and David here. The libraries should be reviewed with great caution and less approvers is better here.

Reply:
The list of owners updated to:
```
approvers:
- dims
- deads2k
- liggitt
- soltysh
reviewers:
- dims
- deads2k
- liggitt
- soltysh
```